src/app/forms/custom-sections/psv-brakes/psv-brakes.component.html

```diff
@@ -4,6 +4,7 @@
   [type]="editTypes.AUTOCOMPLETE"
   name="techRecord_brakes_brakeCodeOriginal"
   label="Brake code"
+  [hint]="'If year of manufacture is 1988, check your brake code'"
```

Comment:
is it worth separating this string value out into an enum somewhere or would it only ever get used here?

Reply:
As far as I know, this is the only place this is being used

Reply:
Cool, happy for you to hit merge then 👍